README.md

```diff
@@ -28,6 +28,26 @@ This repository uses [lerna](https://github.com/lerna/lerna) to manage it's pack
 | `react-shopify-app-route-propagator` | [README](packages/react-shopify-app-route-propagator/README.md)                          | [![npm version](https://badge.fury.io/js/%40shopify%2Freact-shopify-app-route-propagator.svg)](https://badge.fury.io/js/%40shopify%react-shopify-app-route-propagator) |
 | `react-shortcuts`                    | [README](packages/react-shortcuts/README.md)                                             | [![npm version](https://badge.fury.io/js/%40shopify%2Freact-shortcuts.svg)](https://badge.fury.io/js/%40shopify%2Freact-html)                                          |
 
+## Development
+
+Running a development version of a quilt package in your project:
```

Comment:
Maybe we should specify this is for when you want to test out local changes in your project?